bundles/org.openhab.binding.modbus.kermi/src/main/resources/OH-INF/thing/xcenter-pvmodulation-channel-types.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="modbus"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<channel-type id="pv-state-channel">
+		<item-type>Contact</item-type>
+		<label>PV Modulation Active</label>
+		<description>State of PV-Modulation</description>
+	</channel-type>
+	<channel-type id="pv-power-channel">
+		<item-type>Number:Watt</item-type>
+		<label>PV Power</label>
```

Comment:
Number:Watt does not exist, i guess you meant Number:Power ?
Each channel should als have the state tag, specify the unit and possibly more.
 
Please see other bindings (like deconz that has both power and energy and there are more). Some common channel types are pre-defined (system channels) , documentation is here:
https://www.openhab.org/docs/developer/bindings/thing-xml.html#system-state-channel-types

Can you check all channel id's as this seem to be a generic issue.

Reply:
I checked all channels (i missed to correct this one) and added Units already (and states, too).